### ISSUE TITLE
CHERI128_DE4_MFS_ROOT fails to build

### ISSUE THREAD

**Issue by arichardson** [CTSRD-CHERI/cheribsd#527]
```
/usr/local/jenkins/workspace/CheriBSD-test-basic/CPU/mips/ISA/hybrid/cheribsd/sys/dev/cfi/cfi_dev.c:290:36: error: converting non-capability type 'u_char *' (aka 'unsigned char *') to capability type 'void * __capability' without an explicit cast; if this is intended use __cheri_tocap
                        error = copyout(&val, rq->buffer++, 1);
                                                        ^
1 error generated.
--- cfi_dev.o ---
```

**Comment by qwattash** [comment]
I remember having fixed this somewhere, perhaps in PR #514 Let me go ahead and merge that so that we can recheck

**Comment by qwattash** [comment]
Should be fixed now
